transhumandesign/kag-base
# [added] Night ambient music

## Status

- **READY**: this PR is (to the best of your knowledge) ready to be incorporated into the game.

## Description

Fixes 2) of #1295.

This adds a new night world ambient music track to WARMusic.as and ChallengeMusic.as.
It is crickets chirping which is common to occur at late afternoon to midnight.
This music plays when `getMap().getDayTime()` is between 0.75 and 1.0 - before sunset starts until before dawn starts.

The music used is the first 45 seconds of this, amplied by +6.1 and saved as .ogg at quality level 6. It is free-to-use: 
https://pixabay.com/sound-effects/cricket-sound-14867/
If you think there is a better soundtrack, let me know.

## Steps to Test or Reproduce

Go to Sandbox or TTH tutorial.
Wait until sunset and night (or type `getMap().SetDayTime(0.7f);` in console to jump right before a sunset).
Notice the night ambient music.
